apps/server/src/controllers/ontimeController.ts

```diff
@@ -455,3 +457,44 @@ export const postNew: RequestHandler = async (req, res) => {
     res.status(400).send({ message: error.toString() });
   }
 };
+
+/**
+ * Retrieves and lists all project files from the uploads directory.
+ * @param req
+ * @param res
+ */
+export const listProjects: RequestHandler = async (_, res: Response<ProjectFileList | { message: string }>) => {
```

Comment:
can you double-check this functions need to be async?

Reply:
hmm, apparently it doesn't, I thought express controllers needed to be async, but guess I was wrong